### ISSUE TITLE
Helpful conversion of an email to a gravatar image URL

### ISSUE THREAD

**Issue by johlrogge** [agical/friendsunhosted#55]
The new profile picture is really cool!

It would be even cooler if F#Un would offer to convert your email-adress to a gravatar image. You still don't have to store the email-address for that, just use it as input to calculate the gravatar hash. right now it was a bit cumbersome to sign in to gravatar just to get the image-link. And lots of non-developers probably don't even think about the possibility of using gravatar.

**Comment by jancborchardt** [comment]
Easier would be to just have a separate input for your email address in the profile tab. That should automatically check for a Gravatar, which can be overridden manually by using the image input.

That would also help when you want to invite friends using the email feature in the friends tab – you could just get a notification that this person already is on friendsunhosted, and then add them.

**Comment by johlrogge** [comment]
Yes, though there are privacy issues with storing email-addresses so we don't want to do that as long as the whole profile is public. I was shooting for a simple way to get the image-URL in case you want to use gravatar without opening a can of privacy worms.

Though it is simple enough to split the profile in a private and public part I thought we don't need to store the email just yet. (doesn't the "this person is already on unhosted" trip on the email privacy issue too?)
